.pre-commit-config.yaml

```diff
@@ -22,7 +22,6 @@ repos:
         description: This hook handles all frontend linting for Kolibri
         entry: yarn run lint-frontend:format
         language: system
```

Comment:
Need to add this regex back if we're passing in filenames
```suggestion
        language: system
        files: \.(js|vue|scss|css)$
```

Reply:
I think we need to go back to the drawing board to figure out how to have cleaner logs in Travis when this hook fails.